openfisca_france/model/prestations/autonomie.py

```diff
@@ -0,0 +1,309 @@
+# -*- coding: utf-8 -*-
+
+
+from __future__ import division
+
+import numpy as np
+from numpy import maximum as max_, minimum as min_, select
+
+from openfisca_france.model.base import *  # noqa analysis:ignore
+
+
+class base_ressources_apa(Variable):
+    column = FloatCol
+    label = u"Ressources considérées dans le calcul de l'APA"
+    entity = Individu
+    definition_period = MONTH
```

Comment:
A-t-on une référence législative pour cette base ressource ?

Reply:
I do not have one at the moment. I would like to merge that PR and incrementally validate those formulas with experts.

Reply:
It might be there (https://www.legifrance.gouv.fr/affichCodeArticle.do;jsessionid=84F4CD20C730823D552B4D904C735C1D.tplgfr35s_3?idArticle=LEGIARTI000032138608&cidTexte=LEGITEXT000006074069&dateTexte=20171002).